checker/manual/troubleshooting.tex

```diff
@@ -683,11 +683,6 @@ \subsection{Build the Type Annotations compiler\label{building-compiler}}
   export PATH=$JSR308/jsr308-langtools/dist/bin:${PATH}
 \end{Verbatim}
 
-You may wish to later put the Checker Framework \<javac> even earlier on
```

Comment:
Are these changes by you, Jeff? Or is this from some other changeset that I didn't have?